ephemeris/shed_install.py

```diff
@@ -384,6 +311,16 @@ def _parser():
                         help="Install tool dependencies through resolver (e.g. conda). "
                              "Will be ignored on galaxy releases older than 16.07. "
                              "Can be overwritten on a per-tool basis in the tools file")
+    parser.add_argument("--latest",
+                        action="store_true",
+                        dest="force_latest_revision",
+                        help="Will override the revisions in the tools file and always install the latest revision.")
+    parser.add_argument("--update",
+                        action="store_true",
+                        dest="update_tools",
+                        help="This updates all tools in the Galaxy to the latest revision. "
```

Comment:
s/the//

Reply:
I'm wondering if we should make this a positional command.

Like `shed_tools update` and `shed_tools install ...`